common-arcana.lic

```diff
@@ -92,7 +92,7 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare')
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
```

Comment:
❓combat-trainer doesn’t look like it’ll pass in the tattoo argument per https://github.com/rpherbig/dr-scripts/blob/596442d3cdb5782cb2b95cf73d01ca80cce2537a/combat-trainer.lic#L1693

Does this mean you plan another PR to that file or am I misreading combat trainer?

Reply:
Combat-trainer would have to be in a separate PR.

Short answer: they have to be separate because common files are not updated in-memory at the same time scripts are.

Breaking changes to common files have to be merged first in a backwards compatible way. Then the change to the consuming script can be merged a few days later, once people (or dependency) have had time to update.

Reply:
Yeah, I've made changes to all of them at the same time and it is a cluster. :P